src/components/_tests_/CommonComponent.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { CommonComponent } from '../CommonComponent';
+import { shallow } from 'enzyme';
```

Comment:
Add into wdio.config into before function next, please
```
import { shallow } from 'enzyme';     
global.shallow = shallow;
```

It is give you opportunity to avoid importing in each file shallow from enzyme.
Try this way, please 

FYI: [example](https://github.com/vladislavkovaliov/react-cdp-for-mentees/blob/master/config/webdriver.config.js#L169)